File: 백준/Bronze/14219. 막대과자 포장/막대과자 포장.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int m = sc.nextInt();
        if (n % 3 != 0 && m % 3 != 0) {
            System.out.println("NO");
        } else {
            System.out.println("YES");
        }
        sc.close();
    }
}

File: 백준/Bronze/14219. 막대과자 포장/막대과자 포장.c
#include <stdio.h>

int main() {
    int n, m;
    scanf("%d %d", &n, &m);
    if (n % 3 && m % 3) {
        printf("NO\n");
    } else {
        printf("YES\n");
    }
    return 0;
}